Adding multiple unique keys (#6438)
dbt-labs/docs.getdbt.com: website/docs/reference/resource-configs/unique_key.md

## Description
A column name or expression that is unique for the inputs of a snapshot. dbt uses this to match records between a result set and an existing snapshot, so that changes can be captured correctly.

> In Versionless and dbt v1.9 and later, [snapshots](/docs/build/snapshots) are defined and configured in YAML files within your `snapshots/` directory. You can specify one or multiple `unique_key`s within the `config` block of your snapshot YAML file.

Reviewer: i'd change it from config block to config key to make it clearer since config block is used in .sql files too

```suggestion
In Versionless and dbt v1.9 and later, [snapshots](/docs/build/snapshots) are defined and configured in YAML files within your `snapshots/` directory. You can specify one or multiple `unique_key` values within your snapshot YAML file's `config` key.
```

</File>

<VersionBlock firstVersion="1.9">

### Use multiple unique keys

> You can configure snapshots to use multiple unique keys for primary_key columns.

Reviewer: ```suggestion
You can configure snapshots to use multiple unique keys for `primary_key` columns.
```

<File name='snapshots/transaction_items_snapshot.yml'>

```yaml
snapshots:
  - name: orders_snapshot
    relation: source('jaffle_shop', 'orders')
    config:
      schema: snapshots
      unique_key: 
        - order_id
        - product_id
      strategy: timestamp
      updated_at: updated_at
      
```

</File>
</VersionBlock>

<VersionBlock lastVersion="1.8">

### Use a combination of two columns as a unique key

> This configuration accepts a valid column expression. As such, you can concatenate two columns together as a unique key if required. It's a good idea to use a separator (e.g. `'-'`) to ensure uniqueness.

Reviewer: ```suggestion
This configuration accepts a valid column expression. As such, you can concatenate two columns together as a unique key if required. It's a good idea to use a separator (for example, `'-'`) to ensure uniqueness.
```
